docs/build/wasp/0.7.0/docs/guide/evm/examples/ERC20.md

```diff
@@ -18,7 +18,7 @@ keywords:
 
 - [Remix IDE](https://remix.ethereum.org/).
 - [A Metamask Wallet](https://metamask.io/).
-- [Connect your MetaMask with the public Testnet](../../chains_and_nodes/testnet.md#interact-with-evm).
+- [Connect your MetaMask with the public Testnet](../../tooling/#metamask).
```

Comment:
Just out of curiosity. This links are not broken, you just want to change it?

Reply:
It's from pages that got outdated but still remain on repo. So, technically not broken but not relevant links anymore.